NetKVM/wlh/ParaNdis6-Impl.cpp

```diff
@@ -923,9 +941,10 @@ tPacketIndicationType ParaNdis_PrepareReceivedPacket(
 #if PARANDIS_SUPPORT_RSC
             if (!(pContext->RSC.bIPv4SupportedQEMU || pContext->RSC.bIPv6SupportedQEMU) && (pHeader->hdr.gso_type != VIRTIO_NET_HDR_GSO_NONE))
             {
-                *pnCoalescedSegmentsCount = PktGetTCPCoalescedSegmentsCount(pPacketInfo, pContext->MaxPacketSize.nMaxDataSize);
+                *pnCoalescedSegmentsCount = PktGetTCPCoalescedSegmentsCount(pContext, pPacketInfo, pHeader->hdr.gso_size);
                 NBLSetRSCInfo(pContext, pNBL, pPacketInfo, *pnCoalescedSegmentsCount, 0);
-                DPrintf(1, "RSC host packet, datalen %d, GSO type %d\n", pPacketInfo->dataLength, pHeader->hdr.gso_type);
+                DPrintf(1, "RSC host packet, datalen %d, GSO type %d, mss %d, %d segments\n",
```

Comment:
Maybe it is a good idea to have an additional patch that raises the level for this print. After it will be printed for each packet.

Reply:
Currently the concept of logging is that in default logging level we print mainly logging of initialization and shutdown, during the connection there are no printouts. If we raise this print, we'll have a lot of such printouts on each file download. This is what I did during debugging and  this created a log of 500 MB in a minute.